sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -212,132 +177,135 @@ def test_get_configuration_setting_label(self, client, appconfiguration_connecti
             and fetched_kv.value == compare_kv.value
             and fetched_kv.content_type == compare_kv.content_type
             and fetched_kv.tags == compare_kv.tags
+            and fetched_kv.label == compare_kv.label
         )
         assert fetched_kv.label is not None
+        client.delete_configuration_setting(key=compare_kv.key, label=compare_kv.label)
 
     @app_config_decorator
-    def test_get_non_existing_configuration_setting(self, client, appconfiguration_connection_string, test_config_setting, test_config_setting_no_label):
-        compare_kv = test_config_setting
+    def test_get_non_existing_configuration_setting(self, appconfiguration_connection_string):
+        client = self.create_client(appconfiguration_connection_string)
+        compare_kv = self.create_config_setting()
+        self.add_for_test(client, compare_kv)
         with pytest.raises(ResourceNotFoundError):
-            client.get_configuration_setting(
-                compare_kv.key, compare_kv.label + "a"
-            )
+            client.get_configuration_setting(compare_kv.key, compare_kv.label + "a")
+        client.delete_configuration_setting(key=compare_kv.key, label=compare_kv.label)
 
     # method: delete_configuration_setting
     @app_config_decorator
-    def test_delete_with_key_no_label(self, client, appconfiguration_connection_string, test_config_setting_no_label):
-        to_delete_kv = test_config_setting_no_label
-        client.delete_configuration_setting(to_delete_kv.key)
+    def test_delete_with_key_no_label(self, appconfiguration_connection_string):
+        client = self.create_client(appconfiguration_connection_string)
+        to_delete_kv = self.create_config_setting_no_label()
```

Comment:
Create or delete?

Reply:
create! I removed the `create` and `add two test config settings` in wrapper.